test/unit/tasks/test_create_forecast.py

```diff
@@ -14,14 +15,28 @@
 
 
 class TestCreateForeCastTask(TestCase):
-    def setUp(self) -> None:
+    @patch("openstef.model.serializer._get_model_uri")
+    def setUp(self, _get_model_uri_mock) -> None:
         self.pj, self.modelspecs = TestData.get_prediction_job_and_modelspecs(pid=307)
         self.serializer = MLflowSerializer(
             trained_models_folder="./test/unit/trained_models"
         )
+
+        # mock model location
+        # Determine absolute location where already stored model is, based on relative path.
+        # This is needed so the model stored in the repo can be found when running remote
+        rel_path = "test/unit/trained_models/mlruns/0/d7719d5d316d4416a947e4f7ea7e73a8/artifacts/model"
+        _get_model_uri_mock.return_value = Path(rel_path).absolute().as_uri()
         # Use MLflowSerializer to load a model
         self.model, _ = self.serializer.load_model(pid=307)
 
+    def test_mocked_model_path(self):
+        """This test explicitely tests if the model path is mocked correctly"""
+        assert (
```

Comment:
Good extra test. It would be even better to test if the path is correctly created in the `MLflowSerializer.load_model` function without mocking all the path creation functions. We then would mock `mlflow.sklearn.load_model`, `self._determine_model_age_from_mlflow_run` and `self._get_model_specs`. Then we can really check if whole path is correctly calculated and set by the `MLflowSerializer.load_model` function

Reply:
Good idea, I added the test